packages/api-client/src/views/Request/types/sidebar-item.ts

```diff
@@ -17,6 +17,8 @@ export type SidebarItem = {
   icon?: string
   edit: (name: string, icon?: string) => void
   delete: () => void
+  documentUrl?: string
+  watchForChanges?: boolean
```

Comment:
Out of scope but the sidebar being nested deep inside  the request view looks wrong. The same sidebar would be shared by tags, collections, and model pages.